projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+
+### Searching existing locations within the entry form
+
+In this solution you directly integrate with the UiTdatabank Search API in your own entry form.
+
+* ✅ Suitable when the locations for the events created through your application are unpredictable in advance
+* ❌ Not recommended when the events created through your application all have the same location
+* ❌ Not recommended when the events created through your application can have a different location, but (1) the amount of possible locations is limited & (2) the list of possible locations is static and won't change a lot over time
+
+#### Example
+
+In the example below, a user in your entry form types the name "Het Depot" in the input field for the location. Under the hood an API call to the UiTdatabank Search API is made to check if that place already exists:
+
+**request**
+
+```
+GET /places/?q=name.\*:"Het Depot" AND address.\*.postalCode:3000
+```
+
+**response**
+
+The response now contains the identifier of "Het Depot":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/place/8248e289-c986-4006-902f-b0616dcbcde7",
+         "@type":"Place"
+      }
+   ]
+}
+```
+
+If the place already exists (like in the example above), it is suggested in the user interface in a human-readable way so that the user can select the place and thus choose to reuse the place for the creation of his or her event.
+
+Only when no results are found, or the results found do not match the place that the user wants to use, a new place can be created by the user.
+
+## Reuse existing organizers
+
+### Storing the organizer ID(s) inside your application
+
+This is the simplest method. You store the identifer(s) of the organizer(s) inside your application.
+
+* ✅ Suitable when all the events created through your application all have the same organizer
+* ✅ Suitable when the events created through your application can have a different organizer, but (1) the amount of organizers is limited & (2) the list of organizers is static and won't change a lot over time
+* ❌ Not suitable when the organizers of the events created through your application are unpredictable in advance
+
+#### Example
+
+In the example below, all the events created through the integration have the same organizer, "Natuurpunt Holsbeek". First you'll need to find the identifier of the organizer `Natuurpunt Holsbeek` in UiTdatabank.
+
+**request**
+
+Perform a `GET` request to the UiTdatabank API including the name of the organizer:
+
+```
+GET /organizers/?q=name.\*:"Natuurpunt Holsbeek"
+```
+
+<!-- theme: warning -->
+
+> For organizers we advice to only search for the `name` and to not include a search on `postalCode` (since this is an optional field for organizers).
```

Comment:
```suggestion
> Contrary to places, for organizers we advice to only search for the `name` and to not include a search on `postalCode` (since this is an optional field for organizers).
```